examples/collections/collections.c

```diff
@@ -165,7 +165,7 @@ int main(int argc, char* argv[]) {
 
   const char* items[] = { "apple", "orange", "banana", "mango", NULL };
 
-  cass_cluster_set_protocol_version(cluster, 2);
+  //cass_cluster_set_protocol_version(cluster, 2);
```

Comment:
Thanks! This could be removed instead of commented (I don't know why that was in there).